### PR TITLE
Add. Init flow

### PR DESCRIPTION
**Pull request by priyajames29**

### CODE REVIEW

File: src/services/InitService.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    },
+  };
+  const logger = LoggingService.getLogger('InitService');
+  logger.debug(JSON.stringify(request));
+  const response = {
+    context: ContextBuilder.getContextWithContext(request.context),
+    message: data,
+  };
+  // eslint-disable-next-line no-promise-executor-return
+  await new Promise((resolve) => setTimeout(resolve, 3000)); // delayed result
```
Comment (hariharanweb) [inline src/services/InitService.js]: No need to delay here. I added delay for search to look real

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/services/InitService.js (hunk 1 above)
Comment (priyajames29) [reply src/services/InitService.js]: It has been removed.

File: src/services/InitService.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+const init = async (request) => {
+  const data = {
+    order: {
+      ...request.message.order,
+      payment: {
+        uri: 'https://api.bpp.com/pay?amt=$amount&txn_id=ksh87yriuro34iyr3p4&mode=upi&vpa=bpp@upi',
+        tl_method: 'http/get',
+        params: {
+          transaction_id: 'ksh87yriuro34iyr3p4',
```
Comment (hariharanweb) [inline src/services/InitService.js]: generate some guid

File: src/services/InitService.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import Api from '../api/Api';
+
+dotenv.config();
+
+const init = async (request) => {
+  const data = {
+    order: {
+      ...request.message.order,
+      payment: {
+        uri: 'https://api.bpp.com/pay?amt=$amount&txn_id=ksh87yriuro34iyr3p4&mode=upi&vpa=bpp@upi',
```
Comment (hariharanweb) [inline src/services/InitService.js:14]: make it bpp url. we could host a dummy site

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/services/InitService.js (hunk 2 above)
Comment (priyajames29) [reply src/services/InitService.js]: uuid is put to create transaction_id

**Review by hariharanweb — APPROVED** [review]

**Pull request merged**